rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/LocalDataTransferServer.java

```diff
@@ -106,7 +106,8 @@ public void run()
                 log.info("Stopping Local Transfer server");
             }
             catch (IOException e) {
-                log.error(String.format("Error starting Local Transfer server %s", e));
+                log.error(String.format("Error starting Local Transfer server %s", e), e);
+                e.printStackTrace();
```

Comment:
It does but just the message. I wanted stack trace but this didnt work out as well. I'll revert this change.